pkg/controller/disruption/disruption.go

```diff
@@ -397,40 +397,40 @@ func (dc *DisruptionController) removeDb(obj interface{}) {
 	if !ok {
 		tombstone, ok := obj.(cache.DeletedFinalStateUnknown)
 		if !ok {
-			klog.Errorf("Couldn't get object from tombstone %+v", obj)
+			klog.ErrorS(nil, "Couldn't get object from tombstone", "obj", obj)
```

Comment:
"With the exception of acronyms like "IP" and the standard "err", don't shorten names."
```suggestion
			klog.ErrorS(nil, "Couldn't get object from tombstone", "object", obj)
```

Reply:
Fixed.